controllers/controllers.js

```diff
@@ -81,5 +83,28 @@ exports.getCommentsC = (req,res,next) => {
     .catch((err) => {
       next(err);
     });
-  
-}
\ No newline at end of file
+};
+
+exports.postCommentsC = (req, res, next) => {
+  const id = req.params.review_id;
+  const { username, body } = req.body;
```

Comment:
Nice use of destructuring here, you could also use destructuring on line 89 to be consistent